ethereum/helpers/uris.ts

```diff
@@ -22,15 +24,15 @@ export async function getURITemplate(
         await normalize(
           `select 
             json_object(
-              'name',case when exists(select * from ${pilotsTable} where rig_id = ${attributesTable}.rig_id and end_time is null) then 'Rig #'||rig_id||' ✈️' else 'Rig #'||rig_id end,
```

Comment:
In here, I updates the metadata queries to use the new lookups table as well as add deal information to the attributes. I made some other changes to just improve the readability and simplicity of the queries.